nemo_aligner/utils/distributed.py

```diff
@@ -217,6 +219,30 @@ def _compute_distributed_log_softmax(vocab_parallel_logits):
     return vocab_parallel_logits - sum_exp_logits.log_().to(vocab_parallel_logits.dtype)
 
 
+def subtract_distributed_logits_with_max(vocab_parallel_logits):
```

Comment:
This function seems useless. Can you double check and remove it if it is useless?